Calculator/MainWindow.xaml.cs

```diff
@@ -54,8 +54,9 @@ public MainWindow()
             symbol = Symbol.None;
             Point = false;
             IsCalculated = false;
-            FirstNumber = "0";
+            FirstNumber = null;
             SecondNumber = null;
+            Screen.Content = "0";
```

Comment:
already defined in line73 in MainWindow.xaml

Reply:
You entered a number at first, 
then you enter the '±',  
and you will find it cannot work successfully, 
**when the program just opened.**
#### You must let the first number equal to **null** instead of '0'.
So I do this yo make it true.
```CSharp
FirstNumber = null;
```
Just like what program has done after entering a 'C'.
---
```CSharp
Screen.Content = "0";
```
It is not the main issue.

Reply:
That‘s OK. I made a misunderstanding. The I question that I wanted to solve is another, but I got it wrong. The main problem is that when you have calculated a result, and you enter '±', you will find it become 0 in your content. I don't know how to solve it. 
Hope you can look at the code, and think how to make it correct.